boards/arm/stm32f4_disco/Kconfig.defconfig

```diff
@@ -17,6 +17,16 @@ config UART_STM32_PORT_2
 
 endif # UART_CONSOLE
 
+if USB_UART_CONSOLE
+
```

Comment:
Don't you need to enable "USB" as well?

Reply:
You are right.
I need USB, USB_DEVICE_STACK but can remove UART_LINE_CTRL.
I'll update the file.